git_machete/github.py

```diff
@@ -28,20 +27,29 @@
 
 
 class GitHubPullRequest(object):
-    def __init__(self, number: int, user: str, base: str, head: str, html_url: str, state: str) -> None:
+    def __init__(self, number: int, user: str, base: str, head: str, html_url: str, state: str, full_repository_name: str, repository_url: str) -> None:
         self.number = number
```

Comment:
While you're here... pls type all these attributes (since I suppose that just typing the corresponding c'tor params is not enough for mypy to capture any type errors in the usages of the attrs)